### PR TITLE
fix(docs): alert on 301, 303 links

### PR DESCRIPTION
**Pull request by 0618**
#### Description of changes

[301](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301), [303](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303) status code should not be acceptable.

Note: We still double check 301 from docs.amplify.aws ([code](https://github.com/aws-amplify/amplify-ui/blob/main/docs/scripts/util/checkLink.ts#L31)) because its broken link returns 301 (https://github.com/aws-amplify/docs/issues/5221). e.g. 'https://www.w3.org/WAI/ARIA/apg/patterns/radtton' returns 404, but 'https://docs.amplify.aws/lib/auth/getarted/q/platform/js' returns 301

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by reesscot — APPROVED** [review]

File: docs/src/pages/[platform]/components/link/index.page.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Link
 description: Link renders an anchor element <a> by default and is primarily used for navigation.
-ariaPattern: https://www.w3.org/WAI/ARIA/apg/patterns/link
+ariaPattern: https://www.w3.org/WAI/ARIA/apg/patterns/link/
```
Comment (zchenwei) [inline docs/src/pages/[platform]/components/link/index.page.mdx:4]: What's the difference between adding a `/` and not adding one?

**Review by zchenwei — APPROVED** [review]
LGTM but tt seems the `docs.amplify.aws` example you are referencing is return 404 now 🥴

File: docs/src/pages/[platform]/components/link/index.page.mdx (hunk 1 above)
Comment (0618) [reply docs/src/pages/[platform]/components/link/index.page.mdx:4]: without `/`, it returns `301` and redirects to the one with `/`

**Comment by 0618** [comment]
> https://docs.amplify.aws/lib/auth/getarted/q/platform/js

Still 301 🥴 

image

**Comment by zchenwei** [comment]
> > https://docs.amplify.aws/lib/auth/getarted/q/platform/js
> 
> Still 301 🥴
> 
> image

Now I see since you asked my last question.

**Pull request merged**